packages/esm-patient-common-lib/src/dashboards/dashboard-extension.scss

```
@@ -0,0 +1,8 @@
+.menu {
+  display: flex;
+  align-items: center;
```

I don't think centering is the right option here. The items should flex from the start regardless of the width of the container.

```suggestion
  align-items: start;
```